### DISCUSSION TITLE
Why are many functions unavailable after the first push

### DISCUSSION THREAD

**Discussion by DowneyFlyfan** [CaiJimmy/hugo-theme-stack#740] [Q&A]
I deployed my blog on Github and directed it to another domain. And  I copied the configuration of the example site. 

For the first time I pushed it to Github, it seems fine. And all things function well. 

Ever since the second time I push my blogs to Github. The "Search" function no longer works. And the switch of "Dark mode" fails whenever I'm not in "Home" page. 

I don't know what's going on. Is there sth. wrong with the configuration of ExampleSite ???

**Comment by DowneyFlyfan** [comment]
By the way, all the functions are available on localhost. 

And, the "mermaid" diagrams sometimes fail and sometimes appear. (I already got that code on hugo official website to my config.)

**Comment by zhixuan2333** [comment]
Do you configure baseurl to your domain name?

**Reply by DowneyFlyfan** [reply]
Yes!!!

**Reply by DowneyFlyfan** [reply]
But it doesn't work

**Reply by CaiJimmy** [reply]
You have typed the wrong domain name.

![image](https://user-images.githubusercontent.com/5889006/209136650-c2c51ec8-ae68-401b-8f1b-e852562e0ca7.png)

**Reply by DowneyFlyfan** [reply]
Thank you ！！！
